DashKit/DashPumpManager.swift

```diff
@@ -37,6 +37,8 @@ open class DashPumpManager: PumpManager {
         return Pod.maximumBasalScheduleEntryCount
     }
     
+    private static let MaxTempBasalGapFillDuration = TimeInterval(seconds: 6)
```

Comment:
[nit] I don't think we capitalize variables, even constants, anywhere else.  Consider `maxTempBasalGapFillDuration`

Reply:
Haha, yeah, slipping back into objective c naming. There are a few other uppercase first letter constants still lingering in our swift code, looks like, in Loop; mainly for notification names.